docs/user_guide/vectors_and_matrices.mdx

```diff
@@ -501,3 +501,69 @@ compile-time.
 ![Fixed resizing](/img/resizing.svg)
 
 </div>
+
+## Matrix Macros 
+### Macros 
+#### dmatrix!
+Construct a dynamic matrix directly from data. The syntax is exactly the same as for matrix!, but instead of producing instances of SMatrix, it produces instances of DMatrix. 
```

Comment:
Maybe use code blocks for vectors, types etc, so that it's consistent with the rest of the user guide. This applies throughout.

Reply:
Updated: Ok I tried to make sure and use code blocks for types, vectors, etc. so I could match the rest of the document.